cvat/apps/documentation/user_guide.md

```diff
@@ -361,11 +435,24 @@ Go to specified frame. Press ``~`` to highlight element.
 ---
 __Open Menu__ button
 
-It is the main menu for the annotation tool. It can be used to download, upload and remove annotations.
+It is the main menu for the annotation tool. It can be used to download, upload and remove annotation.
 
 ![](static/documentation/images/image051.jpg)
 
-As well it shows statistics about the current task, such as:
+Button assignment:
+
+- ``Open Task`` — open task in cvat dashboard
+- ``RunReID Merge`` — combines unbound tracks generated by using autoannotation into one object. [Read more](https://github.com/opencv/cvat/tree/develop/cvat/apps/reid)
```

Comment:
Wrong description for RunReID Merge.

Reply:
Nikita, could you provide right description for this option?

Reply:
You can get the correct description from the link. Also it is better to use relative links. We are going to merge the branch into "master" branch and in this case the link above will still point on develop branch. Could you please check all links as well?